### PR TITLE
Version 2

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

File: manifest.toml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+    [resources.install_dir]
+
+    [resources.permissions]
+    main.url = "/"
+
+    [resources.apt]
+    packages = "coolwsd, code-brand, hunspell-de-de, hunspell-en-gb, hunspell-fr, hunspell"
+    extras.repo = "deb https://collaboraoffice.com/repos/CollaboraOnline/CODE-debian11 ./ "
+    extras.key = "https://keyserver.ubuntu.com/pks/lookup?op=get&search=0x0C54D189F4BA284D 2>/dev/null"
```
Comment (alexAubin) [inline manifest.toml:56]: I think it should be something like this : 

```suggestion
    [resources.apt.extras.collabora]
    packages = "coolwsd, code-brand, hunspell-de-de, hunspell-en-gb, hunspell-fr, hunspell"
    repo = "deb https://collaboraoffice.com/repos/CollaboraOnline/CODE-debian11 ./ "
    key = "https://keyserver.ubuntu.com/pks/lookup?op=get&search=0x0C54D189F4BA284D"
```

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Pull request closed without merging**